R/content-types.R

```diff
@@ -43,6 +43,7 @@ knownContentTypes <- c(
   xlam = "application/vnd.ms-excel.addin.macroEnabled.12",
   xlsb = "application/vnd.ms-excel.sheet.binary.macroEnabled.12",
   feather = "application/feather",
```

Comment:
Officially this has been registered as `application/vnd.apache.arrow.file`: https://www.iana.org/assignments/media-types/application/vnd.apache.arrow.file

Reply:
Feather == Arrow file (see https://arrow.apache.org/faq/); Parquet is its own thing, the `arrow` package happens to include a reader/writer for it. So they would have different MIME types (Parquet doesn't have one yet but I see you've found the jira issue where they're discussing it).

Reply:
excellent suggestion as always @nealrichardson !!